Bash tag adds

BeermotorWB opened the pull request:
Added bash tags for several extant and new plugins.

BeermotorWB committed:
Merge pull request #1 from loot/v0.10

Syncing up fork

BeermotorWB committed:
Bash tag adds

Added Bash tags for several extant plugins and several new entries.

Freso commented on masterlist.yaml:
```diff
@@ -712,3 +721,67 @@ plugins:
         <<: *dirtyPlugin
         itm: 4
         udr: 1
+  - name: 'AyleidCitadel.esp'
```
It would be nice to have the `url` included for new entries as well, allowing other/future editors to easily look up the plugin. :) (Also, I've been talking a bit with [modwat.ch](https://modwat.ch/) about them using the LOOT masterlists to add links to plugins in their lists, which would also need the `url` attributes being set.)

Freso commented on masterlist.yaml:
```diff
@@ -712,3 +721,67 @@ plugins:
+      - Invent
+  - name: 'DBM_HelgenReborn_Patch.esp'
+    tag:
+      - Factions
```
If you can set whatever editor you use to add a new line/line break at the end of the file, that would be great. Not really a big deal, but it prevents commits "fighting" over changing the last line as some editors add it back and others remove it and just clutters diff views unnecessarily.

BeermotorWB replied:
I took it out on purpose actually since I'm used to dealing with yaml on non-Windows platform,  but I'll try to retain the cr/lf at the end from here on.

Freso replied:
(Note that the line breaks will be converted by git when cloning the repository, so while it's `CRLF` for you, it's `LF` for me (on Linux) and will be just `CR` for people on Mac OS <X machines. :) Git does the line ending conversion transparently (unless you explicitly tell it not to!).)

BeermotorWB replied:
Good to know.  In my day job I'm a *nix admin but our desktop machines are all MBPs, so I've been insulated from the Windows CRLF mess for the past decade or so. 

On the Windows gaming box I have at home I'm using Notepad++ which should respect the existing format of the file, but I'm thinking I'll just set it to use LF for YAML files to play it safe. That or I could just use Vim in a WSL terminal like a civilized human.  ;)

Freso reviewed (commented):
Overall LGTM, but I'll leave it for someone more familiar with SSE to merge. :)

BeermotorWB commented:
I think I was on autopilot and forgot about the "delete your fork instead of syncing" recommendation in the documentation. 

I like the branch idea but I didn't want to presume I'd ascended to that branch status yet after only two PRs ;)

Ortham commented on masterlist.yaml:
```diff
@@ -401,6 +404,12 @@ plugins:
+      - C.Owner
+      - C.Regions
+      - C.Water
+      - Graphics
```
I haven't looked at Realistic Water Two in a while, but are C.Location and C.Owner really appropriate?

BeermotorWB replied:
Location yes, but now that you mention it I want to look at C. Owner.

BeermotorWB replied:
Actually @WrinklyNinja since Irma decided she wanted to visit Space Camp after visiting Disney World, I may not get to look at this tonight, especially if we don't have power.

I've edited my fork so I'm going to close this PR and submit a fresh one.

BeermotorWB replied:
OK it's raining sideways here but I figured out the C.Owner tag on RWT: the plugin sets player ownership of a RWT-placed horse trough (RWT_HorseTrough) in the various Hearthfires homes. 

It'll probably be fine to leave it out. If we ever hear of anything clobbering the RWT Horse Troughs we can add it back.

Ortham commented:
My comments would echo Freso's, a trailing newline is standard practice, and providing the `url:` entries for new plugins can be useful (but I myself don't always do so). 

One branch per PR is also generally good practice, and you can then keep the branches from upstream (e.g. `v0.10`) in sync while branching off them to make your PRs, and keep history clean. In this case I can squash and merge, but I prefer PRs to be merged with merge commits, it makes it clear that the source was external.

Ortham commented:
We should probably update the contributing guide to mention this stuff...

BeermotorWB committed:
Tag change for realisticwatertwo

Want to revisit what or why realisticwatertwo.esp is doing as pertains to cell ownership.  Removing c.owner from tags for now.

BeermotorWB committed:
added lf to EOF

Added a linefeed to the end of the file since it was missing.

BeermotorWB commented:
~~Since Irma decided she wanted to go to Space Camp after visiting Disney World, I may not get to look at the realistic water two plugin tonight.~ ~ Irma spared Space Camp and our trees so I was able to get this done. 

I've made a couple of quick changes to my fork (I didn't realize they'd be inherited by this PR) to remove the tag in question and to add a proper LF at the EOF.

Ortham commented:
👍
